src/jdk.javadoc/share/classes/jdk/javadoc/internal/doclint/HtmlTag.java

```diff
@@ -40,20 +40,18 @@
 /**
  * Enum representing HTML tags.
  *
- * The intent of this class is to embody the semantics of W3C HTML 4.01
+ * The intent of this class is to embody the semantics of the current HTML standard,
  * to the extent supported/used by javadoc.
- * In time, we may wish to transition javadoc and doclint to using HTML 5.
  *
- * This is derivative of com.sun.tools.doclets.formats.html.markup.HtmlTag.
+ * This class is derivative of {@link jdk.javadoc.internal.doclets.formats.html.markup.TagName}.
  * Eventually, these two should be merged back together, and possibly made
  * public.
  *
- * @see <a href="http://www.w3.org/TR/REC-html40/">HTML 4.01 Specification</a>
+ * @see <a href="https://html.spec.whatwg.org/multipage/">HTML Living Standard</a>
  * @see <a href="http://www.w3.org/TR/html5/">HTML 5 Specification</a>
- * @see <a href="http://www.w3.org/TR/wai-aria/ ">WAI-ARIA Specification</a>
+ * @see <a href="http://www.w3.org/TR/REC-html40/">HTML 4.01 Specification</a>
+ * @see <a href="http://www.w3.org/TR/wai-aria/">WAI-ARIA Specification</a>
  * @see <a href="http://www.w3.org/TR/aria-in-html/#recommendations-table">WAI-ARIA Recommendations Table</a>
- * @author Bhavesh Patel
- * @author Jonathan Gibbons (revised)
```

Comment:
Thanks for removing those two @author entries. Did they slip through the review of "8235435: Remove (obsolete) @author info from javadoc source and test" or this file was added later?

Reply:
The file was moved here from the `jdk.compiler` module, where we have not removed the `@author` comments.